lib/MetaCPAN/Role/Script.pm

```diff
@@ -10,9 +9,13 @@ use Log::Contextual qw( :log :dlog );
 use MetaCPAN::Model;
 use MetaCPAN::Types qw(:all);
 use MetaCPAN::Queue ();
-use Moose::Role;
+
 use Carp ();
 
+with 'MetaCPAN::Role::HasConfig';
+with 'MetaCPAN::Role::Fastly';
+with 'MetaCPAN::Role::Logger';
```

Comment:
I think we want to prohibit multiple `with`s, unless there's a compelling reason. So, something like:

```perl
with( 'MetaCPAN::Role::Fastly', 'MetaCPAN::Role::HasConfig', 'MetaCPAN::Role::Logger',);
```